crawler/file_processing.py

```diff
@@ -439,6 +456,22 @@ def process_samples(self, add_to_dart: bool) -> None:
         self.backup_file()
         self.create_import_record_for_file()
 
+    def log_unprocessed(self) -> None:
+        """Log the file as unprocessed and ensure it won't be processed in future.
+
+        In this implementation, we assume the reason the file shouldn't be processed is that
```

Comment:
Is it more accurate to say that "we (Sanger) do not want to support unconsolidated files from the centre anymore"? I might have missed some context.